src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/HearingController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.sandl.snlevents.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.UserTransaction;
+import uk.gov.hmcts.reform.sandl.snlevents.model.request.HearingSessionRelationship;
+import uk.gov.hmcts.reform.sandl.snlevents.model.response.HearingInfo;
+import uk.gov.hmcts.reform.sandl.snlevents.repository.db.HearingRepository;
+import uk.gov.hmcts.reform.sandl.snlevents.service.HearingPartService;
+
+import java.util.UUID;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping("hearing")
+public class HearingController {
+
+    @Autowired
+    private HearingRepository hearingRepository;
+
+    @Autowired
+    private HearingPartService hearingPartService;
+
+    @GetMapping(path = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public HearingInfo getHearingById(@PathVariable("id") UUID id) {
+        return new HearingInfo(hearingRepository.findOne(id));
+    }
+
+    @PutMapping(path = "/{hearingId}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity assignHearingPartToSession(
```

Comment:
this method should be in *HearingPart* controller?

Reply:
No it shouldn't, but it won't be moved in this pr. We want to merge it as quick as possible this needs to be done in upcoming stories.

Reply:
Could you then add some comment in order to not forget about it?